### PR TITLE
Tweak: PLG for adding new containers [AI-771]

### PR DESCRIPTION
**Pull request by DennisNerush**

### CODE REVIEW

**Review by matipojo — DISMISSED** [review]

**Review by rotemee — CHANGES_REQUESTED** [review]

File: modules/ai/assets/js/editor/module.js [hunk 1]
```diff
@@ -1,10 +1,63 @@
...
+	onCreateContainer( args ) {
+		if ( args.container.id !== 'document' ) {
+			return;
+		}
+
+		if ( ! shouldShowPromotionIntroduction() ) {
+			return;
+		}
+
+		const element = args.container.view.$el[ 0 ];
```
[the remark this answers is not in the document]
Comment (DennisNerush) [reply modules/ai/assets/js/editor/module.js:29]: it's the enchor

[the remark this answers is not in the document]
Comment (rotemee) [reply modules/ai/assets/js/editor/module.js:27]: @DennisNerush - I meant that maybe you can set the value (`args.container.view.$el[ 0 ]`)  directly in the anchor prop (without declaring the variable).

File: tests/playwright/pages/editor-page.ts [hunk 2]
```diff
@@ -95,6 +95,11 @@ export default class EditorPage extends BasePage {
 		}
 	}
 
+	async hideAiPromotion() {
+		// @ts-expect-error Property 'EDITOR_SESSION_ID' exists on window from ai.
+		window.sessionStorage.setItem( 'ai_promotion_introduction_editor_session_key', window.EDITOR_SESSION_ID );
```
Comment (dmaziar) [inline tests/playwright/pages/editor-page.ts:100]: obviously window is not define in playwright context.
Please use this code from Pro https://github.com/elementor/elementor-pro/blob/dbbb022f0059eef40400e23232b3822714b9ce44/tests/playwright/pages/wp-admin-page.ts#L312
and put it here in core https://github.com/elementor/elementor/blob/fd0c505639dafdf4ad279d4932ab37cd0de8b3d3/tests/playwright/pages/wp-admin-page.ts#L249
We handle pop-ups in this `closeAnnouncementsIfVisible` function

File: tests/elements-regression/tests/elements-regression.test.ts [hunk 3]
```diff
@@ -56,6 +56,7 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 			const helper = new ElementRegressionHelper( page, testInfo );
 			await wpAdminPage.openNewPage();
 			await editorPage.closeNavigatorIfOpen();
+			await editorPage.hideAiPromotion();
```
Comment (dmaziar) [inline tests/elements-regression/tests/elements-regression.test.ts:59]: Please remove this line and read comment below

**Review by AdirD — APPROVED** [review]

**Review by rotemee — APPROVED** [review]

**Pull request merged**